main.tf

```diff
@@ -355,6 +367,24 @@ module "atlantis_sg" {
   tags = merge(local.tags, var.atlantis_security_group_tags)
 }
 
+module "efs_sg" {
+  source  = "terraform-aws-modules/security-group/aws//modules/nfs"
+  version = "v4.8.0"
+  count   = var.enable_ephemeral_storage ? 0 : 1
+
+  name        = "${var.name}-efs"
+  vpc_id      = local.vpc_id
+  description = "Security group allowing access to the EFS storage"
+
+  ingress_cidr_blocks = [var.cidr]
```

Comment:
`var.cidr` can be empty when is reusing existing VPC

Reply:
probably `ingress_cidr_blocks` is unnecessary here because later we have `ingress_with_source_security_group_id`